modules/ROOT/pages/ps-config-fw-rules.adoc

```diff
@@ -0,0 +1,94 @@
+= Configure Firewall Rules to Manage Traffic to a Private Space
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+By default, all traffic to your private space is blocked unless it is explicitly allowed in a firewall rule.
+
+Creating a private space automatically creates these firewall rules:
+
+* Two rules to allow outbound connections from within your private space through ports 8091 and 8092.
```

Comment:
We have different defaults now in CH2.0. I'm pretty sure [this mockup](https://docs.google.com/document/d/1lg5Uic_6mMe0zB46hUa9z5K1oUgI4kliRwHuJe0T1i8/edit#heading=h.1o2q0vn660aj) in the spec is accurate.

Reply:
Updated

Reply:
Updated to match design spec